app/templates/partials/feedback_form.html

```diff
@@ -0,0 +1,28 @@
+<form method="POST" class="feeback__form {{ 'u-hidden' if feedback_form_hidden }}">
+    {{ form.csrf_token }}
+    <div class="group">
+        <div class="feedback__field">
+            <label id="feedback-message-label" class="label venus" for="feedback-message">How can we improve this survey?</label>
+            {% if feedback_form_hidden %}
+            <a class="feeback__close" href="#close">Close</a>
+            {% endif %}
+            <textarea class="input input--textarea js-charlimit-input" name="message" id="feedback-message" rows="8" cols="60" maxlength="2000"></textarea>
+            <div id="feedback-message-remaining" class="input__limit u-mt-xs mercury">Characters remaining: <span>-</span></div>
```

Comment:
That's the same component as we use in the questionnaires so changing the styling will change it across the whole product. Does that give you a preference over padding vs font change?